kpi/views/v2/asset.py

```diff
@@ -121,6 +125,44 @@ class AssetViewSet(
     >
     >       curl -X GET https://[kpi]/api/v2/assets/?collections_first=true&ordering=-name
 
+    <hr>
+
+    Preform bulk actions on assets
+
+    Actions available:
+
+    - `archive`
+    - `delete`
+    - `unarchive`
+    - `undelete` (superusers only)
+
+    <pre class="prettyprint">
+    <b>POST</b> /api/v2/assets/bulk/
+    </pre>
+
+    > Example
+    >
+    >       curl -X POST https://[kpi]/api/v2/assets/bulk/
+
+    > **Payload to preform bulk actions on one or more assets**
+    >
+    >        {
+    >           "payload": {
+    >               "asset_uids": [{string}, ...],
+    >               "action": {string},
+    >           }
+    >        }
+
+    > **Payload to preform bulk actions on ALL assets for authenticated user**
```

Comment:
"perform"

Reply:
more significant edit :joy: let's say "owned by authenticated user" instead of "for authenticated user" just to clarify